gtfs_realtime_translators/translators/path_rail.py

```diff
@@ -109,33 +149,67 @@ def __call__(self, data):
         return FeedMessage.create(entities=entities)
 
     @classmethod
-    def __make_trip_updates(cls, data):
-        trip_updates = []
-        now = int(pendulum.now().timestamp())
+    def __to_unix_time(cls, time):
+        datetime = int(pendulum.from_format(time, 'HH:mm').in_tz(cls.TIMEZONE).timestamp())
+        return datetime
 
-        arrivals = data['results']
-        for arrival in arrivals:
-            arrival_updates = arrival['messages']
-            for idx, update in enumerate(arrival_updates):
-                try:
-                    route_id = cls.ROUTE_ID_LOOKUP[update['lineColor']]
-                    destination = arrival['target']
-                    station = arrival['consideredStation']
+    @classmethod
+    def __route_lookup(cls,service_id, station_shortkey, track_id, destination):
+        service_mapping = cls.SERVICE_LOOKUP.get(str(service_id))
+        station_mapping = cls.STOP_ID_LOOKUP.get(station_shortkey)
+        route_id = service_mapping.get('route_id')
+        headsign = service_mapping.get(destination)
+        stop_id = station_mapping.get("tracks").get(track_id)
+        stop_name = station_mapping.get("stop_name")
+        return {
+           'route_id': route_id,
```

Comment:
Looks like one space lesser than next line? Or it is just git showing that way?

Reply:
It might just be git! my text editor shows the same amount of space